### PR TITLE
Maximum downtime estimate during migration

### PR DESCRIPTION
**Pull request by gally47**
Screenshot 2024-05-08 at 12 36 42

## Motivation and Context

fixes https://github.com/Graylog2/graylog2-server/issues/19270

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: graylog2-web-interface/src/components/datanode/migrations/in-place/JournalDowntimeWarning.tsx [hunk 1]
```diff
@@ -37,7 +37,8 @@ const JournalDowntimeWarning = ({ currentStep, onTriggerStep }: MigrationStepCom
       <h3>Journal downtime size warning</h3>
       <p>Please note that during migration data processing  will stop on your Graylog node, this will result in the journal growing in size.</p>
       <p>Therefore increase your journal volume size.</p>
```
Comment (moesterheld) [inline graylog2-web-interface/src/components/datanode/migrations/in-place/JournalDowntimeWarning.tsx:39]:
```suggestion
      <p>Therefore you might need to increase your journal volume size.</p>
```

File: graylog2-web-interface/src/components/datanode/migrations/in-place/JournalDowntimeWarning.tsx [hunk 2]
```diff
@@ -37,7 +37,8 @@ const JournalDowntimeWarning = ({ currentStep, onTriggerStep }: MigrationStepCom
...
-      <p>Your current journal throughput is: <b>{data} KB/min</b></p>
+      <p>Your current journal size is: <b>{data.journal_size_MB} MB</b> and your current journal throughput is: <b>{data.KBs_per_minute} KB/min</b></p>
+      <p>So your current maximum downtime is: <b>{data.max_downtime_duration}</b></p>
```
Comment (moesterheld) [inline graylog2-web-interface/src/components/datanode/migrations/in-place/JournalDowntimeWarning.tsx:41]:
```suggestion
      <p>Your current maximum downtime for reconfiguring Graylog to point to the data node is: <b>{data.max_downtime_duration}</b></p>
```

**Review by moesterheld — CHANGES_REQUESTED** [review]
looks nice! just some minor textual changes

**Review by moesterheld — APPROVED** [review]
looking good.

**Review by ousmaneo — APPROVED** [review]
LGTM!

**Pull request merged**